File: FundamentosDeLaProgramacionLuisJoyanes4taEdicion/chapter06/CPP/funcAreaRectangulo.cpp
/*Libro:Fundamentos de Programación, Algoritmos, estructura de datos y objetos 4ta edición. 
    Capitulo 06.-Subprogramas (subalgoritmos): Funciones
    Problema 01.-Calcular la superficie (área) de un rectángulo, usando funciones.
    
    Ejemplo1
    Pagina 202
*/
#include<iostream>
using namespace std;

float calculateArea(float base, float altura){
    return base*altura;
}

int main(){
    float base=0, altura=0;

   cout<<"Programa que calcula el area de un rectangulo, usando una funcion:\n\nIngresa la base del rectangulo: ";
   cin>>base;
   cout<<"Ingresa la altura del rectangulo: ";
   cin>>altura;

   cout<<"El area del rectanglo es igual a: "<<calculateArea(base, altura)<<"\n\n";    
}

File: FundamentosDeLaProgramacionLuisJoyanes4taEdicion/chapter06/JAVA/FuncAreaRectangulo.java
/*Libro:Fundamentos de Programación, Algoritmos, estructura de datos y objetos 4ta edición. 
    Capitulo 06.-Subprogramas (subalgoritmos): Funciones
    Problema 01.-Calcular la superficie (área) de un rectángulo, usando funciones.
    
    Ejemplo1
    Pagina 202
*/
import java.util.Scanner;

public class FuncAreaRectangulo{
    static float calculateArea(float base, float altura){
        return base*altura;
    }

    public static void main(String[] args){
        Scanner in=new Scanner(System.in);
        float base=0f, altura=0f;

       System.out.print("Programa que calcula el area de un rectangulo, usando una funcion:\n\nIngresa la base del rectangulo: ");
       base=in.nextFloat();;
       System.out.print("Ingresa la altura del rectangulo: ");
       altura=in.nextFloat();;

       System.out.print("El area del rectanglo es igual a: "+calculateArea(base, altura)+"\n\n");    
    }
}